src/services/messages/snsInvokeMessageProcess.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+// eslint-disable-next-line no-unused-vars
+export const handler = (_event: any, _context: any, _callback: any) => {
```

Comment:
I think you need to make this an async and then await the axios;

```
export const handler = async (.....
  ...
  await axios({....

Reply:
Added!